learning/readme.md

```diff
@@ -0,0 +1,40 @@
+# Learning (Engineering roles)
+
+## Onboarding
+
+Before joining a delivery team, you will spend some time with a mentor focussing on skills relevant to delivery teams.
+
+As a high level summary, we will cover the following -
+
+1. Watch the [introduction to TDD screencast](https://learn.madetech.com/screencasts/tennis.html).
+2. (Pair or mob with a mentor on) [code katas](https://learn.madetech.com/katas) using [Test Driven Development](https://learn.madetech.com/core-skills/tdd/) discipline.
+3. (Pair or mob with a mentor on) subcutaneous acceptance testing, and ATDD discipline.
+4. Explore with a mentor the central architectural concepts of Clean Architecture
+5. Discuss the SOLID principles with a mentor, with reference to Clean Architecture and (A)TDD.
+6. Be introduced to the [Tic Tac Toe sparring exercise](https://learn.madetech.com/sparring/tic-tac-toe/)
+
+## Core Skills
+
+Our Engineering team has built some Core Skills, which are courses around topics valuable to Made Tech.
+
+Currently, each Core Skill is comprised of 3 levels, with acompanying stickers (which you will see on people's laptops).
+
+You earn these stickers by doing coursework, or by passing an assessment, set by members of the Engineering team with expertise in that skill.
+
+### Learn Tech
+
+Every Friday between 13:30 and 17:00 the team come together to focus on Core Skills.
+
+There are two teams, each with their own standup. Details can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1ZUCmchTJnIymhUn_B_iZIYPLPz476PfaZ5KlZ_Gjd1s/edit#gid=27390006) along with a record of assessments.
```

Comment:
Is including an internal spreadsheet necessary? Is it useful in the open source handbook to do so? Suggest replacing with something like 'Full details can be found in the topic of #learntech on Slack'

Reply:
Fair enough - I wasn't really considering the handbook as an internal 'resource' but I can see the logic.